tokenizer.go

```diff
@@ -101,6 +105,15 @@ func uintVecToSlice(arrPtr *C.uint, len int) []uint32 {
 	return slice
 }
 
+func offsetVecToSlice(arrPtr *C.struct_Tuple, len int) []Offset {
+	arr := unsafe.Slice(arrPtr, len)
```

Comment:
who is responsible for releasing this native struct?

Reply:
@daulet indeed that was a leak, I fixed it by doing things differently, i.e., avoiding the tuple struct thing alltogether and just flattening the offset vector returned by the Rust tokenizer, passing that to the go side like the other vectors (only difference is this vector is of length 2*len), and recombining it on the go side to a slice of offsets. See last commit.